README.rst

```diff
@@ -17,8 +17,8 @@
 .. |Nightly wheels| image:: https://github.com/scikit-learn/scikit-learn/workflows/Wheel%20builder/badge.svg?event=schedule
 .. _`Nightly wheels`: https://github.com/scikit-learn/scikit-learn/actions?query=workflow%3A%22Wheel+builder%22+event%3Aschedule
 
-.. |PythonVersion| image:: https://img.shields.io/badge/python-3.7%20%7C%203.8%20%7C%203.9-blue
-.. _PythonVersion: https://img.shields.io/badge/python-3.7%20%7C%203.8%20%7C%203.9-blue
+.. |PythonVersion| image:: https://img.shields.io/badge/python-3.7%20%7C%203.8%20%7C%203.9%20%7C%203.10-blue
+.. _PythonVersion: https://docs.python.org/3
```

Comment:
Thanks for the feedback @rth.

I'm not 100% sure I follow. I think what you're saying is:
* You agree with my change away from the previous link, since linking out to [an image of the badge itself](https://img.shields.io/badge/python-3.7%20%7C%203.8%20%7C%203.9%20%7C%203.10-blue) isn't particularly useful.
* You disagree with the replacement link I've proposed.

Assuming that's the case, I'd be happy to just remove the link or replace it with something else suitable. But first, I'll just explain my rationale. The thought was that the `black` badge points to information about `black`, so maybe the Python link could point to information about Python. A use-case could be a potential `sklearn` user trying to find a supported Python version that works for them. Clicking through, they'd be one link away from the [comparison of Python minor versions](https://docs.python.org/3/whatsnew/index.html).

With that said, I'd be happy to change it however you see fit. Just let me know :)

Reply:
I think pointing to pypi is a good alternative. This is what some libraries do such as [scrapy](https://github.com/scrapy/scrapy) or [requests](https://github.com/psf/requests).

```suggestion
.. _PythonVersion: https://pypi.org/project/scikit-learn/
```

In this sense, the versions displayed here directly corresponds to the [wheels](https://pypi.org/project/scikit-learn/#files) we have on pypi.

Reply:
Cool, that works. I hadn't linked to PyPI because the next badge does so, but no harm in the two badges linking to the same destination.